website/src/components/CodeSnippet/index.tsx

```diff
@@ -28,9 +29,11 @@ interface CodeSnippetProps {
   snippet: string;
 }
 
-export const CodeSnippet: React.FC<CodeSnippetProps> = ({ snippet, title }) => {
+export const CodeSnippet: React.FC<CodeSnippetProps> = ({ snippet, title}) => {
+  const {isDarkTheme} = useThemeContext()
+
   return (
-    <div className="snippet">
+    <div className={`snippet ${!isDarkTheme && 'whiteCodeBlock'}`}>
```

Comment:
This will lead to `snippet false` if the condition fails. That `false` probably shouldn't be here

Reply:
Also it appears that this condition is false during the initial rendering, even on white theme

See the deployed test. When refreshing the website, the snippet starts with a grey background.
Then changing to dark theme and back to light theme, the background is correctly white